docs/de/docs/index.md

```diff
@@ -214,54 +208,54 @@ INFO:     Application startup complete.
 </div>
 
 <details markdown="1">
-<summary>About the command <code>uvicorn main:app --reload</code>...</summary>
+<summary>Über den Befehl <code>uvicorn main:app --reload</code>...</summary>
 
-The command `uvicorn main:app` refers to:
+Der Befehl `uvicorn main:app` bezieht sich auf:
 
-* `main`: the file `main.py` (the Python "module").
-* `app`: the object created inside of `main.py` with the line `app = FastAPI()`.
-* `--reload`: make the server restart after code changes. Only do this for development.
+* `main`: die Datei `main.py` (das Python-"Modul").
+* `app`: das Objekt, das innerhalb von `main.py` mit der Zeile `app = FastAPI()` erzeugt wurde.
+* `--reload`: lässt den Server nach Codeänderungen neu starten. Tun Sie dies nur während der Entwicklung.
```

Comment:
`Tun`is not so nice german, maybe use `Aktivieren Sie dies nur während der Entwicklung`, another alternative would be `Benutzen`.

Reply:
Or: Aktivieren Sie diese Option nur während der Entwicklung.